tests/test-general.php

```diff
@@ -91,12 +110,85 @@ public function test_archiveless_is_method() {
 		$this->assertFalse( Archiveless::is( get_post( $this->archiveable_post ) ) );
 	}
 
-	public function test_always_included_outside_of_main_query() {
+	public function test_always_included_outside_of_main_query_by_default_with_wp_query() {
+		$post_ids = $this->query(
+			[
+				'fields'           => 'ids',
+				'posts_per_page'   => -1,
+				'suppress_filters' => false,
+			]
+		);
+
+		$this->assertContains( $this->archiveless_post, $post_ids );
+		$this->assertContains( $this->archiveable_post, $post_ids );
+	}
+
+	/**
+	 * Test that archiveless posts are included in get_posts() calls by default.
```

Comment:
```suggestion
	 * Test that archiveless posts are not included in get_posts() calls by default.
```
I think I'm reading everything right...